src/org/labkey/test/components/DomainFieldRow.java

```diff
@@ -346,7 +373,7 @@ protected ElementCache newElementCache()
 
     public static class DomainFieldRowFinder extends WebDriverComponentFinder<DomainFieldRow, DomainFieldRowFinder>
     {
-        private final Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "domain-field-row");
+        private final Locator.XPathLocator _baseLocator = Locator.tagWithClassContaining("div", "domain-field-row");
```

Comment:
`tagWithClassContaining` shouldn't be necessary here. `tagWithClass` doesn't care about extra spaces or additional classes.

Reply:
I did it this way because when there's a warning or error, the class changes to literally domain-field-row-warning or domain-field-row-error- it's not adding a new class, changing the existing one.